spec/beaker/hypervisor/openstack_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'fog'
+
+module Beaker
+  describe OpenStack do
+
+    let(:options) { make_opts.merge({'logger' => double().as_null_object}) }
+
+    let(:openstack) {
+      stub_const("Fog::Compute", MockComputeClient)
+      stub_const("Fog::Network", MockNetworkClient)
```

Comment:
@azhurbilo if you checkout this [puppet_helpers_spec code](https://github.com/puppetlabs/beaker/blob/master/spec/beaker/dsl/helpers/puppet_helpers_spec.rb#L778-L781), you can see an example of how we do it.  A better way would be to have a delegator method so you don't have to mock so deeply, but stub instead.  But if you're just looking to get a test for this going quickly I would do something like this for that line:
```ruby
    mock_servers = Object.new
    allow( mock_servers ).to receive( :create )
    allow( @compute_client ).to receive( :servers ).and_return( mock_servers )
```
or potentially change the create line to an expect if that's what you're testing
```ruby
    expect( mock_servers ).to receive( :create ).with( options_answer )
```
let me know if any of this is unclear, and we can discuss it further if needed.